test/Redistribution.test.ts

```diff
@@ -62,6 +63,22 @@ const overlay_4 = '0xaedb2a8007316805b4d64b249ea39c5a1c4a9ce51dc8432724241f41ecb
 const nonce_4 = '0xb5555b33b5555b33b5555b33b5555b33b5555b33b5555b33b5555b33b5555b33';
 const depth_4 = '0x06';
 
+let node_5: string;
+const overlay_5 = '0x141680b0d9c7ab250672fd4603ac13e39e47de6e2c93d71bbdc66459a6c5e39f';
+const stakeAmount_5 = '100000000000000000';
+const nonce_5 = '0xb5555b33b5555b33b5555b33b5555b33b5555b33b5555b33b5555b33b5555b33';
+const hash_5 = '0xb5555b33b5555b33b5555b33b5555b33b5555b33b5555b33b5555b33b5555b33';
+const depth_5 = '0x06';
+const reveal_nonce_5 = '0xb5555b33b5555b33b5555b33b5555b33b5555b33b5555b33b5555b33b5555b33';
+
+let node_6: string;
+const overlay_6 = '0x152d169abc6e6a0e0a2a7b78dcfea0bebe32942f05e9bb10ee2996203d5361ef';
+const stakeAmount_6 = '100000000000000000';
+const nonce_6 = '0xb5555b33b5555b33b5555b33b5555b33b5555b33b5555b33b5555b33b5555b33';
+const hash_6 = '0xb5555b33b5555b33b5555b33b5555b33b5555b33b5555b33b5555b33b5555b33';
+const depth_6 = '0x06';
+const reveal_nonce_6 = '0xb5555b33b5555b33b5555b33b5555b33b5555b33b5555b33b5555b33b5555b33';
```

Comment:
why was it necessary to add 2 more node definitions? why does not your test work with the already existing nodes?

Reply:
Because I wanted to test for a particular number of rounds that are skipped, and with those rounds current nodes overlay won't work.  Anchor will be out of depth for them as you probably know.
So I took the rounds I wanted to check for and found addresses and overlays that would work. 
It could be done differently with the same nodes but then I would need to mine a lot of rounds to get to those matching anchors, which would also slow down tests for this case but wouldn't test it any better than the current setup.

Reply:
that's why I let you know the two other approaches in the main comment of the review.

Reply:
That just seemed like the much harder way to achieve the thing I wanted :)

I could use depth 0x00 but that seems like cheating the test and even 0x01 is not into the anchor I wanted to use. 
I don't want to change the anchor and mine 1000 blocks to find some matching anchor to current nodes as I am also trying to replicate some real-world scenario where a few rounds would be skipped, not thousands so I took that as constraints.

Let's say I want to use the same depth of 0x06 as the rest of the tests. I have an anchor that starts with "0xa92b32" 
How would you adjust the nonce of current nodes to match that anchor?

Reply:
it is not harder way IMO but cleaner for sure. we shouldn't add nodes for all new cases.

in what particular point your test cases depend on the depth to call 0 depth a cheat?

I use https://github.com/rndlabs/swarm-tools-rs/pull/7 for creating nonces

Reply:
Because if we use depth 0x00 then we are not properly testing proximity and anchors/overlays which is part of the test.

Reply:
will close this for now, If I make the time I will revisit this as it's not actually a problem, and it's more of tooling improvement which is of low importance.